data/en/mbs_0111.json

```diff
@@ -44,14 +44,16 @@
                     "type": "Basic",
                     "id": "use-of-information",
                     "content": [{
+                        "description": "<div class=\"panel panel--simple panel--info\"><div class=\"panel__body\"><h2 class=\"venus\">Coronavirus (COVID-19) additional guidance </h2><p><b>If you have not traded for all, or some, of the period</b>: still select “yes, you can provide figures” and enter turnover, even if this is “0”.<br><b>Exclude</b>: furlough payments received in figures.<br><b>Explain figures</b> in the comments section to minimise us contacting you and to help us tell an industry story.<br></p></div></div>",
+
                         "list": ["Data should relate to all sites in England, Scotland and Wales unless otherwise stated.", "You can provide informed estimates if actual figures aren\u2019t available.", "We will treat your data securely and confidentially."]
                     }]
                 }],
                 "preview_content": {
                     "id": "preview",
                     "title": "Information you need",
                     "content": [{
-                        "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
+                        "description": "<a rel='noopener noreferrer' target='_blank' href='https://www.ons.gov.uk/surveys/informationforbusinesses/businesssurveys/monthlybusinesssurveyproductionandservices'>View the survey information and questions before you start the survey (external link)</a>"
```

Comment:
Both of these, I think, need to be there. You can add another object to the content array with the link in. Just put the link as the first item and the text as the second.